graphql/graphiql
# peer resolutions and upgrades

- don't allow < 0.12 anymore because graphql-language-service-utils depends on `ExecutableDefinitions` (see https://github.com/graphql/graphiql/commit/895e68537fd802b8b6ddf2578a1f76f85982c773)
- Fix peer resolutions,
- a clone of graphql/codemirror-graphql#259
- resolves  #863
- minor version updates to test and production dependencies
- add `regeneratorRuntime` as a polyfill for CDN bundle only
- ensure `dist` build is a more progressive build
- ~~add `common-shakify` to tree shake and reduce bundle size~~ (dynamic requires downstream in the interface caused a global eject :()
